python/hail/expr/tests.py

```diff
@@ -856,4 +856,76 @@ def test_gp_dosage(self):
         self.assertAlmostEqual(eval_expr(functions.gp_dosage([0.0, 1.0, 0.0])), 1.0)
         self.assertAlmostEqual(eval_expr(functions.gp_dosage([0.0, 0.0, 1.0])), 2.0)
         self.assertAlmostEqual(eval_expr(functions.gp_dosage([0.5, 0.5, 0.0])), 0.5)
-        self.assertAlmostEqual(eval_expr(functions.gp_dosage([0.0, 0.5, 0.5])), 1.5)
\ No newline at end of file
+        self.assertAlmostEqual(eval_expr(functions.gp_dosage([0.0, 0.5, 0.5])), 1.5)
+
+    def test_call(self):
+        c2_homref = functions.capture(Call([0, 0]))
+        c2_het = functions.capture(Call([1, 0], phased=True))
+        c2_homvar = functions.capture(Call([1, 1]))
+        c2_hetvar = functions.capture(Call([2, 1], phased=True))
+        c1 = functions.capture(Call([1]))
+        c0 = functions.capture(Call([]))
+        cNull = functions.capture(functions.null(TCall()))
```

Comment:
I don't think you need to capture a null call (tho it doesn't hurt).

Reply:
I wanted to test that all of the methods on a null call including ploidy and phased return None. This is further down in the file.